pkg/prompt/import.go

```diff
@@ -38,16 +39,16 @@ func handleImportFile(prompterApp WalletPrompterInterface, filePath string) (*wa
 	}
 
 	// Validate nickname
-	if !wallet.NicknameIsValid(account.Nickname) {
+	if !account.NicknameIsValid(acc.Nickname) {
```

Comment:
What about providing a wallet.CheckAccount(acc) function that will perform all checks ?

Reply:
good idea, I'll try to do that in the following PR on wallet package